src/components/vm/disks/diskAdd.jsx

```diff
@@ -513,11 +514,49 @@ export class AddDiskModalBody extends React.Component {
             break;
         }
         case 'file': {
+            this.setState({ file: value });
```

Comment:
Please add this to the `this.state =` initialization in the ctor, so that it has a complete schema there.

(I know this is just moving code, but while we are at it..)